hibernate-core/src/main/java/org/hibernate/boot/model/internal/PropertyBinder.java

```diff
@@ -292,8 +292,9 @@ private void callAttributeBinders(Property property, Map<String, PersistentClass
 								: persistentClasses.get( holder.getEntityName() );
 				binder.bind( containingAnnotation, buildingContext, persistentClass, property );
 			}
-			catch ( Exception e ) {
-				throw new AnnotationException( "error processing @AttributeBinderType annotation '" + containingAnnotation + "'", e );
+			catch (Exception e) {
+				throw new AnnotationException( "error processing @AttributeBinderType annotation '" + containingAnnotation + "' for attribute '"
+						+ property.getName() + "' of class '" + holder.getEntityName() + "'", e );
```

Comment:
Why not this?

```suggestion
				throw new AnnotationException( "error processing @AttributeBinderType annotation '" + containingAnnotation + "' for attribute " + qualify( holder.getPath(), name ), e );
```

Reply:
Did not know / notice that usage, that is all. Should I change it? Most other usages of qualify in that class use property instead of attribute, should I change the wording for my change back again as well?

```java
throw new AnnotationException( "error processing @AttributeBinderType annotation '" + containingAnnotation + "' for property '"
		+ qualify( holder.getPath(), name ) + "'", e );
```

Reply:
> Should I change it? 

Yes.

> Most other usages of qualify in that class use property instead of attribute, should I change the wording for my change back again as well?

Yes, that seems reasonable.  If you are willing.  I agree with @gavinking  in that I prefer `attribute` - its the phrase JPA uses.  But I'm fine with either; it should just be consistent.